modules/hal_gigadevice/Kconfig

```diff
@@ -1,6 +1,11 @@
 # Copyright (c) 2021 ATL-Electronics
 # SPDX-License-Identifier: Apache-2.0
 
+config GD32_SOC_SERIES
+	string
+	default SOC_SERIES if SOC_FAMILY_GD32
+	default "gd32vf103" if SOC_SERIES_RISCV_GIGADEVICE_GD32VF103
```

Comment:
I see others like NEORV32 using `SOC_SERIES_NEORV32`, so why can't we just use `SOC_SERIES_GD32VF103`, and set `SOC_SERIES` to `gd32vf103`? I think this is overcomplicating the SoC Kconfig part.

Reply:
My feeling is that RISC-V should be all reviewed in future to better accommodate all needs. I think we are spending too much here right now. Currently, we have a solution (not the best) which works without problems and it is made in our side, which is easy to handle.

My proposal is move forward with which we have and contribute  in a future discussions to reorganize riscv with our experience. We have a edge case right now and this could be good for start the improvement but I think it is better get in.

Reply:
I will change it after dev-review discussion.